lib/assets/stylesheets/modules/_technical-documentation.scss

```diff
@@ -226,16 +200,20 @@
     p+p, p+ul, p+ol,
     ul+p, ul+ol,
     ol+p {
-      margin-top: $gutter-one-third;
+      margin-top: govuk-spacing(2);
     }
   }
 
   hr {
```

Comment:
I think this is duplicating / overriding hr styles on line 74.

https://github.com/alphagov/tech-docs-gem/pull/97/files#diff-d7f12a4697fd2195a8c62339caee5c77R74-R77